tests/src/test/scala/common/Wsk.scala

```diff
@@ -71,16 +71,6 @@ import whisk.utils.retry
 
 case class WskProps(
     authKey: String = WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting),
-    cert: String = WhiskProperties.getFileRelativeToWhiskHome("ansible/roles/nginx/files/openwhisk-client-cert.pem").getAbsolutePath,
```

Comment:
done, only question I have is about the apihost, they don't seem too different, but which one should I keep?